scripts/ai/update-metadata.ts

```diff
@@ -75,6 +90,15 @@ async function processFile(file: string) {
 
     processedFiles++;
 
+    // TODO: Determine if we need to skip certain tile types like placeholders.
```

Comment:
@scytacki Do we want to include "Placeholder" as a type in `tileTypes`?

Reply:
I think it is fine to leave them in, and we can just filter them out in the UI. I'll remove the comment.